assets/js/googlesitekit/widgets/components/WidgetContextRenderer.js

```diff
@@ -33,7 +33,7 @@ import { Grid, Row, Cell } from '../../../material-components';
 const { useSelect } = Data;
 
 const WidgetContextRenderer = ( props ) => {
-	const { slug, className, Header, Footer } = props;
+	const { id, slug, className, Header, Footer } = props;
```

Comment:
@kuasha420, please, add the new `id` prop to the component's propTypes list.

Reply:
@eugene-manuilov oops, coming from the TypeScript land, I keep forgetting this. Fixed!